websites/B/Brain.fm/presence.ts

```diff
@@ -0,0 +1,108 @@
+const presence = new Presence({
+		clientId: "981442534167441428"
+	}),
+	browsingTimestamp = Math.floor(Date.now() / 1000);
+
+presence.on("UpdateData", () => {
+	const presenceData: PresenceData = {
+			largeImageKey: "logo",
+			startTimestamp: browsingTimestamp
+		},
+		{ pathname, hostname } = document.location;
+
+	if (["www.brain.fm", "try.brain.fm"].includes(hostname)) {
+		if (["/", "/home"].includes(pathname))
+			presenceData.details = "Viewing the home page";
+		else if (pathname.startsWith("/about"))
+			presenceData.details = "Viewing the about page";
+		else if (pathname.startsWith("/pdfs"))
+			presenceData.details = "Reading a PDF";
+		else if (pathname.startsWith("/science"))
+			presenceData.details = "Viewing the science page";
+		else if (pathname.startsWith("/press"))
+			presenceData.details = "Viewing the press page";
+		else if (pathname.startsWith("/pricing"))
+			presenceData.details = "Viewing pricing options";
+		else if (pathname.startsWith("/jobs"))
+			presenceData.details = "Viewing the jobs page";
+		else if (pathname.startsWith("/faq"))
+			presenceData.details = "Viewing the FAQ page";
+		else if (pathname.startsWith("/contact"))
+			presenceData.details = "Viewing the contact page";
+		else if (pathname.startsWith("/terms"))
+			presenceData.details = "Viewing the terms page";
+		else if (pathname.startsWith("/privacy"))
+			presenceData.details = "Viewing the privacy page";
+		else if (pathname.startsWith("/legal"))
+			presenceData.details = "Viewing the legal page";
+		else if (pathname.startsWith("/forgot-password"))
+			presenceData.details = "Resetting their password";
+	} else if (hostname.startsWith("my")) {
+		if (pathname === "/") {
+			if (
+				document.querySelector(
+					"#root > div > div.sc-gKsewC.sc-iBPRYJ.sc-gGiJkG.efCouF.cOWINi.jzdkxv"
+				)
+			)
+				presenceData.details = "Logging in";
+			else if (
+				document.querySelector("#root > div > div > div > div.sc-jsNsAp.byZiWf")
+			)
+				presenceData.details = "Choosing mode";
+		} else if (pathname.startsWith("/signup"))
+			presenceData.details = "Signing up";
+		else if (pathname.startsWith("/welcome"))
+			presenceData.details = "Getting started";
+		else if (["/focus", "/relax", "/sleep"].includes(pathname)) {
+			switch (pathname) {
```

Comment:
this is redundant, please pick either one of these

Reply:
If you look farther down, the songTypeInfo, presence details, large image key, small image key and small image key text are all the same. So I figured this would be more effecient. Let me know if you still want a change, and if so, please clarify what exactly you would like that change to be.

Reply:
@ImRodry I'm waiting on clarification on this and then I think this presence is good to go.

Reply:
Its set up this way because, as I stated previously, the songTypeInfo, presence details, large image key, small image key and small image key text are all the same. It would be redundant to repeat the code for that in each if statement as you are suggesting. I am resolving this conversation.